client/.storybook/manager.js

```diff
@@ -0,0 +1,6 @@
+import { addons } from '@storybook/addons';
```

Comment:
This file wasn't generated by the script. I created it as part of the custom theming.

Ref: https://storybook.js.org/docs/react/configure/theming#create-a-theme-quickstart